[v5] Changes to Configuration - CacheOptions (Config #2) (#7697)
AzureAD/microsoft-authentication-library-for-js: lib/msal-browser/docs/caching.md

    - previous failed request 
    - performance data

> > :bulb: Temporary cache entries will always be stored in session storage or in memory. MSAL will fallback to memory storage if local/session storage is not available. Please read the warning below for more information.

Reviewer: ```suggestion
> :bulb: Temporary cache entries will always be stored in session storage or in memory. MSAL will fallback to memory storage if sessionStorage is not available.
```

> :bulb: The authorization code is only stored in memory and will be discarded after redeeming it for tokens.

## Warning :warning:

**NOTE: `temporaryCacheLocation` is deprecated as of MSAL v5 and will be removed in a future release.**

> Overriding `temporaryCacheLocation` should be done with caution. Specifically when choosing `localStorage`. Interaction in more than one tab/window will not be supported and you may receive `interaction_in_progress` errors unexpectedly. This is an escape hatch, not a fully supported feature.

Reviewer 2: We are keeping the docs? I assumed if it is removed, wouldn't it be just removal notice in migration guidance since this branch is V5?

Author: Thanks! This is now removed as it was deprecated for v4 and removed in v5.
